src/styles/main.scss

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: calc((#{$day-size} * 7) + (#{$gap} * 6));
```

Comment:
Ensure that the variables `$day-size` and `$gap` are defined. They are used in calculations and need to be set for the SCSS to compile correctly.